src/features/authorization/managers/registerManager.ts

```diff
@@ -24,10 +24,13 @@ export const useRegisterManager = () => {
     email: Yup.string()
       .email(t('signup:incorrectEmail'))
       .required(t('signup:required')),
-    password: Yup.string()
-      .min(8, t('signup:passwordLengthMin'))
-      .max(64, t('signup:passwordLengthMax'))
-      .required(t('signup:required')),
+    password: Yup.string()ß
+      .min(8, 'Password must be at least 8 characters long')
+      .max(64, 'Password can be up to 64 characters long')
+      .required('Required field')
+      .test('no-spaces', 'Password cannot contain only spaces', (value) => {
```

Comment:
We should block not also passwords with only spaces, but also any space in password. We should use regex expression here that will allow only specific signs.

Reply:
Sounds good will take a look later today thanks !